app/assets/javascripts/draw-forfeit.js

```diff
@@ -0,0 +1,138 @@
+$(function() {
+  $(".request-draw").click(function() {
+    var updateURL = $(event.currentTarget).data("update-url");
+    var color = $(event.currentTarget).data("color");
+    if (color == "black") {
+      $.ajax({
+        type: 'PUT',
+        url: updateURL,
+        dataType: 'json',
+        data: { game: { black_draw: true } },
+        success: function(data) {
+          alert("A draw has been requested!");
+          $.ajax({
+            type: 'PUT',
+            url: Routes.game_draw_path(game_id)
```

Comment:
I"m not sure how the `Routes.game_draw_path(game_id)` works in js. but I did get an error for game_id. if i hardcoded the game id in there, might be easier just to throw the id in a data-attr and grab it from there like you are on the updateURL.